tests/ApplicationTestCase.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\Typo3DocsTheme;
+
+use phpDocumentor\Guides\Bootstrap\DependencyInjection\BootstrapExtension;
+use phpDocumentor\Guides\Cli\DependencyInjection\ApplicationExtension;
+use phpDocumentor\Guides\Cli\DependencyInjection\ContainerFactory;
+use phpDocumentor\Guides\DependencyInjection\TestExtension;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension;
+
+class ApplicationTestCase extends TestCase
```

Comment:
Should we make this class "abstract" (like TestCase is also abstract)? So it is clear that is is meant for extending?

Reply:
we could but we should do that with the original test case

Reply:
So the ApplicationTestCase can run as a test?

Reply:
no, you are right, made it abstract